Tests/TestFrameworks/TUnit.Specs/TUnit.Specs.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <RootNamespace>TUnit.Specs</RootNamespace>
+    <AssemblyName>TUnit.Specs</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Src\FluentAssertions\FluentAssertions.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="TUnit" Version="0.1.817" />
```

Comment:
I don't expect a lot..the API has been fairly stable for a while but I cannot guarantee it wouldn't change since I'm trying to get community feedback at the moment. So the API could adapt if it provided a better experience. If that's a problem we can put this on hold.

That being said, the only logic fluent assertions really uses is the assembly and the assertion exception fully qualified type. I don't really see those changing.

Reply:
I agree